packages/amplify-category-api/resources/schemas/appsync/AppsyncCLIInputs.schema.json

```diff
@@ -0,0 +1,289 @@
+{
```

Comment:
This file looks like it was renamed. Please remove this one that doesn't seem relevant anymore